pkg/kubefed2/federate/enable.go

```diff
@@ -308,7 +309,8 @@ func typeConfigForTarget(apiResource metav1.APIResource, federateDirective *Fede
 			ComparisonField:    spec.ComparisonField,
 			PropagationEnabled: true,
 			Placement: fedv1a1.APIResource{
-				Kind: fmt.Sprintf("Federated%sPlacement", kind),
+				Kind:       fmt.Sprintf("Federated%sPlacement", kind),
+				PluralName: fmt.Sprintf("federated%splacement", pluralName),
```

Comment:
The template should reuse the target's plural name in composing it's plural name, but placement doesn't need to.

Reply:
My bad, I removed the wrong line in the last commit. I just pushed a new commit that pluralizes only `typeConfig.Spec.Template.PluralName` using the API resource name.